tests/staggered_invert_test_gtest.hpp

```diff
@@ -107,6 +107,9 @@ TEST_P(StaggeredInvertTest, verify)
   auto ca_basis_tmp = inv_param.ca_basis;
   if (solve_type == QUDA_DIRECT_SOLVE && inverter_type == QUDA_CA_GCR_INVERTER) inv_param.ca_basis = QUDA_POWER_BASIS;
 
+  // Single precision needs a tiny bump due to small host/device precision deviations
+  if (prec == QUDA_SINGLE_PRECISION) verify_tol *= 1.03;
```

Comment:
Is this still needed after my recent PR?

Reply:
Yeah, just tried it